includes/class-ess-ajax.php

```diff
@@ -78,7 +78,7 @@ public static function get_all_network_shares_count( $page_url = '', $is_ajax_re
 			check_ajax_referer( 'all-network-shares-count', 'security' );
 
 			$page_url = ess_clean( $_POST['page_url'] );
-			$post_id  = intval( $_POST['post_id'] );
+			$post_id  = isset($_POST['post_id']) ? intval( $_POST['post_id'] ) : $post_id;
```

Comment:
Should be like `$post_id  = isset( $_POST['post_id'] ) ? intval( $_POST['post_id'] ) : $post_id;`